saphanabootstrap-formula.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Wed Jun 10 01:49:02 UTC 2020 - Simranpal Singh <simranpal.singh@suse.com>
+
+- Add support to extract zip,rar,exe,sar hana media 
```

Comment:
Please, comment that this change has non backward compatible changes (`hdbserver_extract_dir` changed by `hana_extract_dir`)